vignettes/UsingSelfControlledCohort.Rmd

```diff
@@ -0,0 +1,222 @@
+
+The time between the start of the observation period and Time 0 is called the washout period and a minimum allowed washout period can be set when defining the analysis. Similarly the time between Time 0 and the observation period end date is the followup period. A minimum followup period can also be set.
+
+A common type of exposure is exposure to a drug but an exposure could be any set of characteristics that are captured in your data. We will look at a few different ways to define exposures.
+
+# Tables used in the analysis
+
+Conceptually there are two primary tables used in the analysis along with a few supporting tables that are part of the OMOP Common Data Model
+
+```{r, echo=F}
+exposure_table <- data.frame(field    = c("person_id", "exposure_concept_id", "exposure_start_date", "exposure_end_date"),
+                             datatype = c("INTEGER",    "INTEGER",             "DATE",               "DATE"))
+
+outcome_table <- data.frame(field    = c("person_id", "outcome_concept_id", "outcome_start_date"),
+                            datatype = c("INTEGER",   "INTEGER",            "DATE"))
+
+knitr::kable(exposure_table)
+knitr::kable(outcome_table)
+```
+
+
+These tables can be created manually or standard CDM tables can be used (eg. drug_era => exposure_table, condition_era => outcome_table). Every exposure-outcome combination in these tables will be included in the analysis unless otherwise specified. The additional CDM tables that are required for this analyis are `observation_period` (for observation start and end dates) and `person` (for year of birth).
+
+# A first example
+
+The SelfControlledCohort package makes it easy to run examine the association between every drug and every condition in a CDM database. The Eunomia package provides a mini CDM that runs entirely within R that we can use for examples.
+
+
+```{r}
+# A first example with
+library(SelfControlledCohort)
+library(Eunomia)
+
+connectionDetails <- Eunomia::getEunomiaConnectionDetails()
+
+result <- runSelfControlledCohort(
+  connectionDetails = connectionDetails,
+  cdmDatabaseSchema = "main",
+  exposureIds = '', # inlcude all drugs as exposures
+  outcomeIds = '' # include all conditions as outcomes
+)
+
+library(dplyr)
+result$estimates %>% 
+  arrange(desc(irr)) %>% 
+  head()
+```
+
+The result of our analysis is a dataframe with one row per exposure-outcome pair along with all the relevant statistics. Let's interpret the resluts for amoxicillin exposure (concept ID 1713332) and the outcome of Chronic sinusitis (concept ID 257012)
```

Comment:
You go in to the details later, but perhaps list all off the statistics that  are computed here before giving the concrete examples.

Reply:
I added a section that describes the output statistics at the beginning of the vignette.